### PR TITLE
Fixes #2859 - Remove addon link for mobile Firefox users

### LINKED ISSUE #2859: Firefox for Android addon offered to Fenix users on webcompat.com
**URL**: https://webcompat.com/

**Browser / Version**: Firefox Mobile 68.0
**Operating System**: Android 8.0.0
**Tested Another Browser**: Yes

**Problem type**: Something else
**Description**: Web Compat on top left of it's homepage shows Fenix users a link to download the extension on AMO, although Fenix doesn't support extensions.
**Steps to Reproduce**:

<details>
<summary>Browser Configuration</summary>
<ul>
  <li>None</li>
</ul>

</details>

_From [webcompat.com](https://webcompat.com/) with ❤️_

### PR DESCRIPTION
**Pull request by miketaylr**
Fennec is basically EOL, and Fenix doesn't yet support arbitrary
addons in the release channel (as far as I know...). So let's just
remove the prompt for mobile users entirely, rather than parse the
UA version and offer the addono for whatever existing Fennec users
there are.

### CODE REVIEW

File: webcompat/templates/nav/addon-links.html [hunk 1]
```diff
@@ -41,9 +32,10 @@
...
     <li class="nav-item">
-        <a class="nav-link" href="https://github.com/webcompat/webcompat.com/issues?state=open" title="Navigate to GitHub issues">
+        <a class="nav-link" href="https://github.com/webcompat/webcompat.com/issues?state=open"
+            title="Navigate to GitHub issues">
             <span class="link-text">Give Feedback</span>
         </a>
     </li>
     {%- endif -%}
-</ul>
+</ul>
```
Comment (miketaylr) [inline webcompat/templates/nav/addon-links.html:41]: (i guess my vscode did some auto-formatting, let me know if you want me to revert these unrelated style changes)

**Review by karlcow — CHANGES_REQUESTED** [review]
You need to remove or change the associated tests with the addon link.

File: webcompat/templates/nav/addon-links.html (hunk 1 above)
Comment (karlcow) [reply webcompat/templates/nav/addon-links.html:41]: You are a terrible person. 👹

**Comment by miketaylr** [comment]
Oh yeah, thanks -- should have run the tests first.

File: webcompat/templates/nav/addon-links.html (hunk 1 above)
Comment (miketaylr) [reply webcompat/templates/nav/addon-links.html:41]: (i mean, we already knew that...)

**Review by karlcow — APPROVED** [review]
❤️
